curation-migration-datasync/database/dataService.integration.ts

```diff
@@ -0,0 +1,125 @@
+import { truncateDynamoDb } from '../dynamodb/dynamoUtilities';
+import { dbClient } from '../dynamodb/dynamoDbClient';
+import sinon from 'sinon';
+import * as SecretManager from '../secretManager';
+import { writeClient } from './dbClient';
+import { Knex } from 'knex';
+import { DataService } from './dataService';
+import { CuratedFeedProspectItem } from '../types';
+
+describe('database integration test', function () {
+  //aim of this test is to validate knex integration and assumptions.
+  //use cases will be tested in index.integration.ts
+  let db: Knex;
+  beforeAll(async () => {
+    await truncateDynamoDb(dbClient);
+    sinon.stub(SecretManager, 'getDbCredentials').resolves({
+      readHost: 'localhost',
+      readUsername: 'root',
+      readPassword: '',
+      writeHost: 'localhost',
+      writeUsername: 'root',
+      writePassword: '',
+      port: '3310',
```

Comment:
We can use environment variables here for the credentials. This is why CI can't hit the mysql database at all.